snmp/datadog_checks/snmp/utils.py

```diff
@@ -34,17 +34,27 @@ def get_profile_definition(profile):
     return profile['definition']
 
 
-def _get_profiles_root():
+def _get_profiles_confd_root():
     # type: () -> str
     # NOTE: this separate helper function exists for mocking purposes.
     confd = get_config('confd_path')
     return os.path.join(confd, 'snmp.d', 'profiles')
 
 
+def _get_profiles_site_root():
+    # type: () -> str
+    here = os.path.dirname(__file__)
+    return os.path.join(here, 'data', 'profiles')
+
+
 def _read_profile_definition(definition_file):
     # type: (str) -> Dict[str, Any]
     if not os.path.isabs(definition_file):
-        definition_file = os.path.join(_get_profiles_root(), definition_file)
+        definition_conf_file = os.path.join(_get_profiles_confd_root(), definition_file)
+        if not os.path.isfile(definition_conf_file):
+            definition_file = os.path.join(_get_profiles_site_root(), definition_file)
+        else:
+            definition_file = definition_conf_file
```

Comment:
(Nit) This logic grows a bit more complex than initially. Should we extract it as a separate helper?

```python
def resolve_profile_definition_path(path):
    # type: (str) -> str
    if os.path.isabs(path):
        return path

    path_in_confd = os.path.join(_get_profiles_confd_root(), path)
    if os.path.isfile(path_in_confd):
        return path_in_confd

    return os.path.join(_get_profiles_site_root(), path)


def _read_profile_definition(definition_file):
    path = resolve_profile_definition_path(definition_file)
    ...
```

Reply:
Done.